user-edit.css

```diff
@@ -34,4 +34,24 @@
 
 .two-factor-methods-table tbody tr:nth-child(odd) {
 	background-color: #f9f9f9;
-}
\ No newline at end of file
+}
+
+@media print {
+	body * {
+		visibility: hidden;
+		max-height: 100px;
```

Comment:
Possibly a hidden overflow could work with max-height, or simply use an explicit height and absolute positioning on all elements?